### ISSUE TITLE
Incorrect refactoring: use-assigned-variable

### ISSUE THREAD

**Issue by nathanaelhuffman** [sourcery-ai/sourcery#66]
### Issue description or question
original code:
```
 def write_n(self, addr, data):
        if isinstance(addr, str):  
            if addr.endswith("L"):
                addr = addr[:-1]
            addr = int(addr, 16)
        orig_addr = addr
        for dat in data:
            self.write32(addr, dat)
            addr += 4

        return orig_addr, data
```
Suggested refactor: (note that addr increments by 4 each loop in the original) so write32 occurs at different addresses each loop,  but in the refactored version all writes occur at orig_addr which doesn't increment.
```
    def write_n(self, addr, data):
        if isinstance(addr, str):
            if addr.endswith("L")
                addr = addr[:-1]
            addr = int(addr, 16)
        orig_addr = addr
        for dat in data:
            self.write32(orig_addr, dat)
            addr += 4

        return orig_addr, data
```
### Sourcery Version
0.7.6

### Code editor or IDE name and version

vscode

### OS name and version

Windows

**Comment by Hellebore** [comment]
Fix identified - will be in the next release.

**Comment by Hellebore** [comment]
resolved with release of 0.7.9
